src/dbnode/config/m3dbnode-local-etcd.yml

```diff
@@ -63,7 +63,7 @@ db:
     bootstrappers:
         - filesystem
         - commitlog
-        - noop-none
+        - uninitialized
```

Comment:
Hm as before, do we want to be putting peers after `commitlog` in all these configurations?

Also very surprised that we don't have peers actually listed here by default.

Reply:
yeah sure thing. Probably because it was too confusing to explain it to people haha